include/swift/SIL/AbstractionPattern.h

```diff
@@ -1020,6 +1126,19 @@ class AbstractionPattern {
   /// it.
   AbstractionPattern getReferenceStorageReferentType() const;
 
+  // SWIFT_ENABLE_TENSORFLOW
+  /// Given that the value being abstracted is a function type, return the
+  /// abstraction pattern for the derivative function.
+  ///
+  /// The arguments are the same as the arguments to
+  /// `AnyFunctionType::getAutoDiffDerivativeFunctionType()`.
+  AbstractionPattern getAutoDiffDerivativeFunctionType(
+      IndexSubset *indices, unsigned resultIndex,
+      AutoDiffDerivativeFunctionKind kind,
+      LookupConformanceFn lookupConformance,
+      GenericSignature whereClauseGenericSignature = GenericSignature(),
```

Comment:
Minor: the `whereClauseGenericSignature` argument of `AnyFunctionType::getAutoDiffDerivativeFunctionType` has been renamed to `derivativeGenericSignature` [on `master`](https://github.com/apple/swift/blob/fa1544974b67bcfd2c1cba1023cfb3a381ce7133/include/swift/AST/Types.h#L3260):
```suggestion
      GenericSignature derivativeGenericSignature = GenericSignature(),
```
The renaming will soon propagate to `tensorflow` branch.

Reply:
done